lib/logstash/outputs/syslog.rb

```diff
@@ -138,11 +152,11 @@ def receive(event)
 
     if @is_rfc3164
       timestamp = event.sprintf("%{+MMM dd HH:mm:ss}")
-      syslog_msg = "<#{priority.to_s}>#{timestamp} #{sourcehost} #{appname}[#{procid}]: #{event.sprintf(@message)}"
+      syslog_msg = "<#{priority.to_s}>#{timestamp} #{sourcehost} #{appname}[#{procid}]: #{message}"
```

Comment:
this was added in https://github.com/logstash-plugins/logstash-output-syslog/pull/12